do.py

```diff
@@ -175,14 +170,10 @@ def build_c_integration_tests():
     expand_file('code-experiments/src/coco.h',
                 'code-experiments/test/integration-test/coco.h',
                 {'COCO_VERSION': git_version(pep440=True)})
-    copy_file('code-experiments/src/bbob2009_testcases.txt',
-              'code-experiments/test/integration-test/bbob2009_testcases.txt')
-    copy_file('code-experiments/src/bbob2009_testcases2.txt',
-              'code-experiments/test/integration-test/bbob2009_testcases2.txt')
-    make("code-experiments/test/integration-test", "clean",
-         verbose=_build_verbosity)
-    make("code-experiments/test/integration-test", "all",
-         verbose=_build_verbosity)
+    copy_file('code-experiments/src/bbob2009_testcases.txt', 'code-experiments/test/integration-test/bbob2009_testcases.txt')
```

Comment:
sorry, my comment was regarding PEP 8 and _incommented_ code, otherwise line breaks are OK